aspnetcore/includes/RP/sql.md

```diff
@@ -2,10 +2,12 @@
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
-The `MovieContext` object handles the task of connecting to the database and mapping `Movie` objects to database records. The database context is registered with the [Dependency Injection](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method in the *Startup.cs* file:
+The `MovieContext` object handles the task of connecting to the database and mapping `Movie` objects to database records. The database context is registered with the [Dependency Injection (DI)](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method in the *Startup.cs* file:
 
 [!code-csharp[](code/Startup.cs?name=snippet2&highlight=6-8)]
 
+See [Using DbContext with Dependency Injection](https://docs.microsoft.com/en-us/ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection) for details on using `DbContext` with DI.
```

Comment:
- Use the following relative link instead: /ef/core/miscellaneous/configuring-dbcontext#using-dbcontext-with-dependency-injection
- Using DbContext with Dependency Injection --> Using DbContext with DI
- Rephrase this sentence to "For more information on using `DbContext` with DI, see ...". This is the structure we recently decided to adopt. It's the sentence structure recommended by an internal tool we use.